.github/workflows/ci.yml

```diff
@@ -22,17 +22,48 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Download dependencies
-        run: composer update --no-interaction --prefer-dist --optimize-autoloader --prefer-stable
+        run: composer update --no-interaction --optimize-autoloader --prefer-stable
 
       - name: Run tests
         run: ./vendor/bin/simple-phpunit
 
+  build8:
+    name: Build PHP 8
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      matrix:
+        php: ['8.0', '8.1']
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+          tools: flex
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Mimic PHP 8.0
+        run: composer config platform.php 8.0.999
+        if: matrix.php > 8
+
+      - name: Download dependencies
+        run: composer update --no-interaction --optimize-autoloader --prefer-stable
+
+      - name: Run tests
+        run: ./vendor/bin/simple-phpunit
+        env:
+          SYMFONY_PHPUNIT_VERSION: 9.5
```

Comment:
Can we let the phpunit bridge decide the version?

Reply:
No, because it chooses 9.4 which causes the tests to fail on PHP 8.1.

Reply:
This change just forces it to use the latest PHPUnit 9.x on PHP 8, which is reasonable IMO.

Reply:
FYI, this will be fixed in the next Symfony releases: https://github.com/symfony/symfony/pull/43200